extensions/azurecore/src/azureResource/providers/databaseServer/databaseServerService.ts

```diff
@@ -31,6 +30,7 @@ export class AzureResourceDatabaseServerService extends ResourceServiceBase<DbSe
 			loginName: resource.properties.administratorLogin,
 			defaultDatabaseName: 'master',
 			subscriptionId: resource.subscriptionId,
+			subscriptionName: resource.subscriptionName,
```

Comment:
Hm, not really a fan of having multiple fields for the same object (a subscription). Would rather have subscription be an object 
```
subscription: { 
   id: resource.subscriptionId,
   name: resource.subscriptionName
}
```

makes it easier to work with I think

Reply:
I made a subscription object for AzureResource interface, but unfortunately the GraphData and interfaces extending it represent the data returned by the Azure Api call, I am keeping it as is.